pages/apim/3.x/kubernetes/apim-kubernetes-operator-installation-local.adoc

```diff
@@ -0,0 +1,247 @@
+* link:https://curl.se/[curl^] in case you do not have it already - to check, type `curl -V` in your command-line tool
+* link:https://k3d.io/#installation[k3d^]
+* A standard command-line tool of your choice
+* Optionally, link:https://k9scli.io/topics/install/[k9s^] for monitoring and managing your k8s cluster and pods
+
+NOTE: The GKO GitHub repository is public and available link:https://github.com/gravitee-io/gravitee-kubernetes-operator[here^]. Cloning it is not a requirement for this deployment scenario so this is just for your reference.
+
+
+== STEP 1: Create a local cluster
+
+To create a local APIM-ready k3d Kubernetes cluster, run the following command:
+
+....
+curl -i https://raw.githubusercontent.com/gravitee-io/gravitee-kubernetes-operator/chore-inline-k3d-script/scripts/k3d.sh | bash
+....
+
+NOTE: This operation may take a few minutes to complete.
+
+Make note of the endpoints listed at the end of the command-line output:
+
+....
+(...)
+
+Available endpoints are:
+
+    Gateway       http://localhost:9000/gateway
+    Management    http://localhost:9000/management
+    Console       http://localhost:9000/console/#!/login
+
+To update APIM components (e.g. APIM Gateway) to use a new docker image run:
+
+> docker tag <image> k3d-graviteeio.docker.localhost:12345/graviteeio/apim-gateway:3.19
+> docker push k3d-graviteeio.docker.localhost:12345/graviteeio/apim-gateway:3.19
+> kubectl rollout restart deployment apim-apim3-gateway
+....
+
+
+== STEP 2: Check if the Gravitee CRDs are available on your cluster
+
+Run the following command:
+
+....
+kubectl get crd
+....
+
+The command-line output should include `apidefinitions.gravitee.io` and `managementcontexts.gravitee.io`, as shown in the example below:
+
+....
+NAME                              CREATED AT
+addons.k3s.cattle.io              2022-09-28T10:25:02Z
+helmcharts.helm.cattle.io         2022-09-28T10:25:02Z
+helmchartconfigs.helm.cattle.io   2022-09-28T10:25:02Z
+apidefinitions.gravitee.io        2022-09-28T10:34:38Z
+managementcontexts.gravitee.io    2022-09-28T10:34:38Z
+....
+
+== STEP 3: Monitor the APIM pods startup until they are ready
+
+The pods will take some time to start and must become ready before you can deploy the GKO (described in the next step).
+
+To monitor from the command line, run the following command:
+
+....
+kubectl wait --for=condition=ready pod -l app.kubernetes.io/name=apim3 --timeout 360s
+....
+
+After some time, the command-line output should look similar to the example below:
+
+....
+pod/apim-apim3-api-7b8b85d79-b7hkh condition met
+pod/apim-apim3-ui-5997c5d5c7-r9xdj condition met
+pod/apim-apim3-gateway-66cbd5c9d9-tv5jz condition met
+....
+
+You can increase the timeout if your machine is slower. If you encounter an error, you will not be able to proceed to the next steps - please contact Gravitee for assistance with this.
+
+NOTE: Alternatively, you can monitor your cluster and pods using link:https://k9scli.io/[K9s].
```

Comment:
I'm not sure if it's useful to talk about K9s in the doc. k9s is the tool I use. @a-cordier use vsCode plugin and @Okhelifi intellij 😁

Reply:
Agree - removed mentions:)